engo.go

```diff
@@ -118,6 +116,16 @@ func Run(o RunOptions, defaultScene Scene) {
 
 	opts = o
 
+	// TODO: We may need to check this later on for platform
+	InitGl()
```

Comment:
Yes, I understand but this way you can specify on your project what to do.
Engo is a `lib` you should be able to set the project how you want it and maybe you have some more stuff to do in a `init`.